sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactDataView.java

```diff
@@ -99,7 +99,9 @@ protected void initView(String params) {
 		editComponent = ControllerProvider.getContactController()
 			.getContactDataEditComponent(getContactRef().getUuid(), ViewMode.NORMAL, contactDto.isPseudonymized());
 
-		addCreateFromCaseButtonLogic();
+		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.CASE_SURVEILANCE)) {
```

Comment:
Should check CASE_CREATE right too, please use `UiUtil.isPermitted`

Reply:
updated